src/components/careers/MostTrusted/index.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react'
+import styled from 'styled-components'
+
+const Section = styled.section`
+  width: 100%;
+  height: 100vh;
+  background-color: ${(p) => p.theme.palette.white};
+
+  @media screen and (max-width: 768px) {
```

Comment:
This `max-width` is consistently appearing across the app. I think it would be wise to extract it.

Reply:
It is indeed used repeteadly. We'll look into it when putting everything together in the final layout